src/__test__/components/data-management/AnalysisModal.test.jsx

```diff
@@ -86,83 +97,76 @@ describe('AnalysisModal', () => {
     component.find(Button).at(0).simulate('click', eventStub);
     component.update();
 
+    // Launch button should be enabled
+    expect(component.find(Button).at(2).props().disabled).toEqual(false);
+
     // Name should be changed
     expect(component.find(EditableField).first().text()).toEqual(newProjectName);
+    expect(onLaunchSpy).toBeCalledTimes(0);
+    expect(onCancelSpy).toBeCalledTimes(0);
   });
 
-  it('Shows a loader if experiment data is not available', () => {
```

Comment:
This test is not necessary -- if experiment data is not available, we will never get to render the AnalysisModal component